src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -410,24 +411,31 @@ private function email(Document $provider): ?EmailAdapter
         $credentials = $provider->getAttribute('credentials');
         return match ($provider->getAttribute('provider')) {
             'mock' => new Mock('username', 'password'),
-            'mailgun' => new Mailgun($credentials['apiKey'], $credentials['domain'], $credentials['isEuRegion']),
+            'smtp' => new SMTP(
+                $credentials['host'],
+                $credentials['port'],
+                $credentials['username'],
+                $credentials['password'],
```

Comment:
These can be unset. Won't that be a problem since the SMTP adapter requires a string?